File: C#/1006 - Média 2.cs
/* 
   ==UserScript==
 @name         Média 2
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1006 - Média 2
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/

using System;

class URI {
    
  static void Main(string[] args) {
      
    double valueA = Convert.ToDouble(Console.ReadLine());
    double valueB = Convert.ToDouble(Console.ReadLine());
    double valueC = Convert.ToDouble(Console.ReadLine());
    
    double media = (valueA*2 + valueB*3 + valueC*5)/10;
    
    Console.WriteLine("MEDIA = {0:F1}", media);
    
  }
}

File: Java/1006 - Média 2.java
/* 
   ==UserScript==
 @name         Média 2
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1006 - Média 2
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/

import java.util.Scanner;
import java.io.IOException;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner scan = new Scanner(System.in);
		double notaA = scan.nextDouble();
		double notaB = scan.nextDouble();
		double notaC = scan.nextDouble();
		
		double media = (notaA * 2 + notaB * 3 + notaC * 5)/10;
		
		System.out.println(String.format("MEDIA = %.1f", media));
 
    }
 
}
